frontend/src/pages/articles/components/article-card/article-card.tsx

```diff
@@ -71,7 +71,7 @@ const ArticleCard: React.FC<Properties> = ({ article, user }) => {
           </li>
         </ul>
         <Icon iconName="share" className={styles.icon} />
-        <Link to={AppRoute.ROOT} className={styles.readMore}>
+        <Link to={AppRoute.ARTICLE} className={styles.readMore}>
```

Comment:
You should pass :id to open actual article by route

Reply:
I'm working on it in this task: https://github.com/BinaryStudioAcademy/bsa-2023-writorium/issues/152

Reply:
@AnastasiaR2  I know - and this comment was written week ago, but this PR is still not merged

Reply:
Just for @IrynaK-a  to know that she doesn't need to do this part now